paddle/fluid/platform/cpu_helper.cc

```diff
@@ -29,6 +29,12 @@ namespace platform {
 
 void SetNumThreads(int num_threads) {
 #ifdef PADDLE_USE_OPENBLAS
+// windows has no support for openblas multi-thread
```

Comment:
Please paste the openblas issue here, otherwise other people will be confused why it was disabled.

Reply:
ok